src/neuroglancer/ui/tool.ts

```diff
@@ -45,20 +45,22 @@ export class ToolActivation<ToolType extends Tool = Tool> extends RefCounted {
   bindInputEventMap(inputEventMap: EventActionMap) {
     this.inputEventMapBinder(inputEventMap, this);
   }
+  cancel() {
+    this.tool.layer.manager.root.toolBinder.deactivate_();
```

Comment:
Can you add a check that `this == toolBinder.activeTool` to prevent spurious canceling?

Reply:
`toolBinder.activeTool` is private to `ToolBinder`, should I make that public-with-an-underscore too?

Reply:
yeah that would be fine.

Reply:
I have made that change.